about/index.md

```diff
@@ -6,9 +6,32 @@ layout: default
+**Collaboration fuels our culture.**
 
-LLNL produces software on a daily basis. Some of this software is used only internally; other components are licensed for use by external partners and collaborators; and still other software is released, or even actively developed, in the open on software hosting platforms such as GitHub.com, Bitbucket.org, Sourceforge.net, and others.
+Welcome to the Lawrence Livermore National Laboratory (LLNL) software portal&mdash;a hub for open source software (OSS) produced at the Lab.
+
+LLNL developers create and evolve software on a daily basis. Some of this software is used only internally; other components are licensed for use by external partners and collaborators; and still other software is released and actively developed out in the open on software-hosting platforms. This website is the gateway to all of it.
+
+LLNL is a Department of Energy (DOE) institution, so we abide by the [Federal Source Code Policy](https://sourcecode.cio.gov) that guides release of open source software (OSS). And not just because we have to but because we *want* to. Long before GitHub was founded, [we started releasing software](https://software.llnl.gov/explore/) for public use and collaboration.
+
+We often work with other national labs, universities, and industry partners. Working on these projects is a lot easier when we can share code directly. For example, building the next generation of supercomputers requires collaboration. The [Exascale Computing Project (ECP)](https://www.exascaleproject.org) uses OSS to develop the exascale ecosystem of apps, analytics, infrastructure, and so much more. The ECP is tackling a huge range of scientific areas, like precision medicine and nuclear physics. Future breakthroughs will likely owe some debt to the best practices and accelerated development made possible by OSS.
+
+One of the great things about a large government organization is that we’re not motivated by profit. “The greater good” might sound like a cliché, but it’s true. LLNL’s national security mission governs our work, so our OSS portfolio of apps, libraries, compilers, and other tools support that mission. This includes everything from monitoring the performance of our supercomputers to making multiphysics codes run more smoothly.
+
+Case in point: The DOE’s [Advanced Simulation and Computing (ASC) Program](https://www.energy.gov/nnsa/missions/maintaining-stockpile) leverages OSS to build, field, and integrate Linux clusters into production service at our [Livermore Computing supercomputing center](https://hpc.llnl.gov/). ASC is responsible for developing computer simulation capabilities that analyze and predict the performance, safety, and reliability of the nation’s nuclear stockpile. In other words, the stakes are high for ensuring our code is accurate. With OSS, we can develop quickly and choose the best solution for meeting our goals.
```

Comment:
@hauten -- Do you think there is a way to work in the ASC open source memo to this paragraph? https://science.energy.gov/~/media/ascr/pdf/research/docs/Doe_lab_developed_software_policy.pdf

Something about how DOE ASC has long advocated for open source software development?